package.json

```diff
@@ -2061,18 +2061,13 @@
         "command": "dotnet.restore.project",
         "title": "%command.dotnet.restore.project%",
         "category": ".NET",
-        "enablement": "dotnet.server.activatedStandalone"
+        "enablement": "dotnet.server.activationContext == 'Roslyn' || dotnet.server.activationContext == 'OmniSharp'"
       },
       {
         "command": "dotnet.restore.all",
         "title": "%command.dotnet.restore.all%",
         "category": ".NET",
-        "enablement": "dotnet.server.activatedStandalone"
-      },
-      {
-        "command": "csharp.downloadDebugger",
```

Comment:
as best as I could tell, this command was never registered anywhere.  cc @WardenGnaw

Reply:
This was part of the original O# extension where users would download the extension then the dependencies would be downloaded since there was no support for platform specific downloads.

However, I think everything is bundled and platform specific this is just an artifact of the past that can be removed 😀